app/graphql/mutations/create_club.rb

```diff
@@ -17,6 +17,8 @@ def resolve(**args)
         invitation_code: rand(100_000..999_999)
       )
 
+      club.users << current_user
```

Comment:
```suggestion
      club = current_user.clubs.create!(
        **args,
        admin: current_user,
        invitation_code: rand(100_000..999_999)
      )
```

Normalement ça rajoute correctement le club à la liste de club de l'utilisateurs sans avoir besoin de la concatenation :muscle: